controllers/default.py

```diff
@@ -345,6 +348,9 @@ def coursechooser():
 @auth.requires_login()
 def removecourse():
     if settings.academy_mode:
+      if not request.args(0):
```

Comment:
Shouldn't this check be outside the `if settings.academy_mode:` clause?

Reply:
yes